### ISSUE TITLE
Adjusting Rect

### ISSUE THREAD

**Issue by raphael-pietrzak** [sloukit/pydew-valley-uzh#35]
When playing the game, the following issues with controls and interactions have been observed:

Unpredictable Tilling Behavior: When using the hoe to till the soil, it is unclear which tile will be tilled. This makes it difficult to adjust the player's position accurately. For instance, if the player needs to move slightly to the left to be in the correct position, the hoe might be used on the left tile, forcing the player to readjust their position unnecessarily.

Inconsistent Rects and Collisions: The collision rectangles (rects) are not properly adjusted. There are also illogical interactions where the player cannot walk over plants. This restriction complicates certain actions, such as watering plants, and causes the player to be awkwardly displaced when planting.

![Screenshot 2024-07-07 at 18 51 28](https://github.com/sloukit/pydew-valley-uzh/assets/93328003/5734f043-8577-45ac-b6d8-19f5e27d15e2) ![Screenshot 2024-07-07 at 18 51 53](https://github.com/sloukit/pydew-valley-uzh/assets/93328003/af17fc21-3f00-4ec1-a729-0a4f2a74f004) ![Screenshot 2024-07-07 at 18 52 09](https://github.com/sloukit/pydew-valley-uzh/assets/93328003/4e93efba-c557-46f3-a0a0-2febe7059ee3)
![Screenshot 2024-07-07 at 18 56 03](https://github.com/sloukit/pydew-valley-uzh/assets/93328003/0e86cdf7-01a6-45b7-b640-f80cfdb6bdf4)

**Comment by michelfinley** [comment]
I'm working on that 😃
The plants have no collisions with the player anymore since #27

**Comment by raphael-pietrzak** [comment]
ow i have already done something 😭😬

**Comment by DangerousVanilla** [comment]
@raphael-pietrzak, @Mega-JC, @michelfinley, can any of you confirm if this is completely resolved based on the intent of this bug? I know there are still active collisions with the small flower type plants around the game which is annoying but I'm not sure if this was supposed to be included here or not. Just figured we could close this out if appropriate.

**Comment by raphael-pietrzak** [comment]
Yes we can close the issue, this was an early bug already fixed, thanks for the catch
